python/src/scipp/compat/mantid.py

```diff
@@ -261,6 +288,30 @@ def convertEventWorkspace_to_dataarray(ws, load_pulse_times):
     return sc.DataArray(**coords_labs_data)
 
 
+def convertMDHistoWorkspace_to_dataset(md_histo):
+    ndims = md_histo.getNumDims()
+    if ndims > 3:
+        raise RuntimeError("Converter cannot process md histo workspace input "
+                           "with > 3 dimensions. Input has {}.".format(ndims))
+    coords = dict()
+    dims_used = []
+    for i in range(ndims):
+        dim = md_histo.getDimension(i)
+        frame = dim.getMDFrame()
+        sc_dim = md_dimension(dim, i)
+        coords[sc_dim] = sc.Variable(dims=[sc_dim],
+                                     values=np.linspace(
+                                         dim.getMinimum(), dim.getMaximum(),
+                                         dim.getNBins()),
```

Comment:
So Mantid is always using uniform bins?

Should this be `nbin + 1`, or is it not actually bin-edges?

Reply:
AFAIK, the MDHistoWorkspace will only support uniform bins.